subsys/bluetooth/Kconfig

```diff
@@ -224,17 +224,34 @@ endif # BT_CONN
 
 
 config BT_ISO
-	bool "Bluetooth Isochronous Channel Support [EXPERIMENTAL]"
+	bool # hidden
+
+config BT_ISO_UNICAST
+	bool "Bluetooth Isochronous Channel Unicast Support [EXPERIMENTAL]"
 	depends on BT_CONN
+	select BT_ISO
 	help
-	  Select this to enable Isochronous Channel support.
+	  This option enables support for Bluetooth Broadcast
+	  Isochronous channels.
+
+# TODO: Split between broadcaster and observer for optimization
+config BT_ISO_BROADCAST
+	bool "Bluetooth Isochronous Channel Broadcast Support [EXPERIMENTAL]"
+	select BT_ISO
+	select BT_EXT_ADV
+	select BT_PER_ADV
+	select BT_PER_ADV_SYNC
+	help
+	  This option enables support for Bluetooth Broadcast
+	  Isochronous channels.
 
 if BT_ISO
 
 config BT_ISO_MAX_CHAN
 	int "Maximum number of simultaneous ISO channels"
 	depends on BT_ISO
-	default BT_MAX_CONN
+	default BT_MAX_CONN if BT_CONN
+	default 1
```

Comment:
Is it allowed to have two default values?  Which takes presedence?  (It seems the intention is that the first should take precedence.)

Reply:
Yes it is. I think it's going from top to bottom, or at the very least any `default x if y` takes precedence.